signature/verifier.go

```diff
@@ -0,0 +1,158 @@
+package signature
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+
+	"github.com/notaryproject/notation-core-go/signer"
+	x509n "github.com/notaryproject/notation-core-go/x509"
+	"github.com/notaryproject/notation-go"
+)
+
+// // maxTimestampAccuracy specifies the max acceptable accuracy for timestamp.
+// const maxTimestampAccuracy = time.Minute
+
+// Verifier verifies artifacts against JWS signatures.
+type Verifier struct {
+	TrustedCerts []*x509.Certificate
+}
```

Comment:
Do we still need this type?

Reply:
Yes to keep existing verification code working we need this. This should be removed when we have implemented trust store.